channels/az.m3u

```diff
@@ -1,25 +1,159 @@
+#EXTINF:-1 tvg-id="ITV.az" tvg-name="ITV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://www.lyngsat-stream.com/logo/tv/ii/ictimai-tv-az.png" group-title="",İctimai TV (480p)
 https://node19.connect.az:8086/ch1/ch1.med.m3u8
+#EXTINF:-1 tvg-id="idmantv.az" tvg-name="İdman TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="http://yoda.az/idmanlive/idman_logo.png" group-title="",İdman TV (1080p) (not 24 7)
+http://str.yodacdn.net/idman/tracks-v1a1/mono.m3u8?token=9cd1fe22fe5a1227a1de1ffc81b11d1f68a57f78-9e60c50db056a1ebab3444b946138d84-1618045991-1618035191
+#EXTINF:-1 tvg-id="interaztv.com" tvg-name="InterAz" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/k51BESZ.png" group-title="",İnterAz TV (1080p)
+http://yayin.netradyom.com:1935/live/interaz/playlist.m3u8
+#EXTINF:-1 tvg-id="interaztv.com" tvg-name="InterAz" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/k51BESZ.png" group-title="",İnterAz TV (1080p)
+rtmp://yayin.netradyom.com/live/interaz
+#EXTINF:-1 tvg-id="interaztv.com" tvg-name="InterAz" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/k51BESZ.png" group-title="",İnterAz TV (720p)
+https://waw3.artiyerelmedya.net/interaz/bant1/chunks.m3u8?hash=d0f574372c36bc44c833002f87764f1d
+#EXTINF:-1 tvg-id="medeniyyettv.az" tvg-name="Medeniyyet TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="http://yoda.az/idmanlive/medeniyyet_logo.png" group-title="",Medeniyyet TV (1080p)
+https://59c7ea6bec1c6.streamlock.net/live/medeniyettv.stream/playlist.m3u8?hash=d0f574372c36bc44c833002f87764f1d
+#EXTINF:-1 tvg-id="medeniyyettv.az" tvg-name="Medeniyyet TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="http://yoda.az/idmanlive/medeniyyet_logo.png" group-title="",Medeniyyet TV (576p)
+https://europe2.livetv.az/azerbaijan/medeniyyet_stream2/playlist.m3u8
+#EXTINF:-1 tvg-id="medeniyyettv.az" tvg-name="Medeniyyet TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="http://yoda.az/idmanlive/medeniyyet_logo.png" group-title="",Medeniyyet TV (576p)
+https://europe2.livetv.az/azerbaijan/medeniyyet_stream2/chunks.m3u8
+#EXTINF:-1 tvg-id="medeniyyettv.az" tvg-name="Medeniyyet TV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="http://yoda.az/idmanlive/medeniyyet_logo.png" group-title="",Medeniyyet TV (404P)
+http://str.yodacdn.net/medeniyyet/tracks-v1a1/mono.m3u8?token=9cd1fe22fe5a1227a1de1ffc81b11d1f68a57f78-9e60c50db056a1ebab3444b946138d84-1618045991-1618035191
+#EXTINF:-1 tvg-id="ntv.az" tvg-name="NTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/D6ni1Vv.png" group-title="",Naxçıvan TV (NTV) (720p)
+https://streams.livetv.az/azerbaijan/nax/playlist.m3u8
+#EXTINF:-1 tvg-id="ntv.az" tvg-name="NTV" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://i.imgur.com/D6ni1Vv.png" group-title="",Naxçıvan TV (NTV) (720p)
+https://streams.livetv.az/azerbaijan/nax/chunks.m3u8
```

Comment:
Both links don't work.

Reply:
This one works for me.

![vlcsnap-2021-04-13-13h31m04s490](https://user-images.githubusercontent.com/30985701/114545696-91aeac00-9c5c-11eb-8160-0d9a79f2ed2b.png)

Maybe not a 24 / 7 channel.

Reply:
Now works for me too. 

If the channel can sometimes not work, it is better to add the label `[not 24/7]` to its name.

Reply:
Both links are down again,
<img width="706" alt="Captura de pantalla 2021-04-13 a las 23 04 38" src="https://user-images.githubusercontent.com/7253922/114614040-b468b100-9cac-11eb-828e-cf95f191dcf2.png">